### PR TITLE
fix: card-mod not working

### PR DESCRIPTION
**Pull request by Gh61**
Hey,

I've created this pull request to fix the functionality issue with the [card-mod](https://github.com/thomasloven/lovelace-card-mod) as mentioned in issue #234.

I've added `style` and `card_mod` option properties. These are required to consist in `config` object for card-mod to read them and create styles.

Please review and provide feedback.
And thank you for your amazing work.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -22,5 +22,9 @@ export default function buildConfig(
     aqi: config.aqi ?? {},
     stats: config.stats ?? [],
     shortcuts: config.shortcuts ?? [],
+
+    // Support for card-mod styling
+    style: config.style,
+    card_mod: config.card_mod,
```
Comment (denysdovhan) [inline src/config.ts:28]: hey, would it work if we just pass these props like this:

```suggestion
    ...config
```

**Comment by Gh61** [comment]
So I pushed changes to pass the remaining properties of config using the spread operator and I've tested it and it's working.

File: src/config.ts [hunk 2]
```diff
@@ -22,5 +22,8 @@ export default function buildConfig(
     aqi: config.aqi ?? {},
     stats: config.stats ?? [],
     shortcuts: config.shortcuts ?? [],
+
+    // Pass other config properties
+    ...config,
```
Comment (denysdovhan) [inline src/config.ts:27]: It's better to spread config before properties, so they won't be overridden by properties from config.

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Good. Please address a small note and let's merge it.

**Comment by Gh61** [comment]
You're totally right. Thank you for code reviewing, this operator is completely new for me.

**Review by denysdovhan — APPROVED** [review]
Good, thank you! Can to please also make this change to vacuum-card?

**Pull request merged**